Course Material/Introduction to Quantum Computing and FiQCI/00 Superposition and measurement.ipynb

```diff
@@ -0,0 +1,682 @@
+    "\n",
+    "To do this, we use **transpile**:\n",
+    "+ Parameters: Circuit to be transpiled and the backend\n",
+    "+ Returns: Circuit with QPU specific instructions\n",
+    "\n",
+    "Below we transpile our circuit for Helmi"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"word-wrap: normal;white-space: pre;background: #fff0;line-height: 1.1;font-family: &quot;Courier New&quot;,Courier,monospace\">global phase: π/2\n",
+       "               ┌──────────────┐┌────────┐┌─┐\n",
+       "      q_0 -> 0 ┤ R(-π/2,-π/2) ├┤ R(π,0) ├┤M├\n",
+       "               └──────────────┘└────────┘└╥┘\n",
+       "ancilla_0 -> 1 ───────────────────────────╫─\n",
+       "                                          ║ \n",
+       "ancilla_1 -> 2 ───────────────────────────╫─\n",
+       "                                          ║ \n",
+       "ancilla_2 -> 3 ───────────────────────────╫─\n",
+       "                                          ║ \n",
+       "ancilla_3 -> 4 ───────────────────────────╫─\n",
+       "                                          ║ \n",
+       "          c: 1/═══════════════════════════╩═\n",
+       "                                          0 </pre>"
+      ],
+      "text/plain": [
+       "global phase: π/2\n",
+       "               ┌──────────────┐┌────────┐┌─┐\n",
+       "      q_0 -> 0 ┤ R(-π/2,-π/2) ├┤ R(π,0) ├┤M├\n",
+       "               └──────────────┘└────────┘└╥┘\n",
+       "ancilla_0 -> 1 ───────────────────────────╫─\n",
+       "                                          ║ \n",
+       "ancilla_1 -> 2 ───────────────────────────╫─\n",
+       "                                          ║ \n",
+       "ancilla_2 -> 3 ───────────────────────────╫─\n",
+       "                                          ║ \n",
+       "ancilla_3 -> 4 ───────────────────────────╫─\n",
+       "                                          ║ \n",
+       "          c: 1/═══════════════════════════╩═\n",
+       "                                          0 "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "qc_transpiled = transpile(qc_measured, backend_helmi)\n",
+    "qc_transpiled.draw()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we see, circuit was mapped to the Helmi 5-qubit QPU. Now that the circuit is transpiled, we can proceed to submit it to the quantum computer."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sixth step: Run circuit on QPU\n",
+    "\n",
+    "Now it is time to submit our circuit to Helmi. \n",
+    "\n",
+    "Quantum comuting is all about probabilities. Because our measurement only yield results in a form of zeros and ones, and each measurement collapses the quantum state, we have to run the circuit multiple times.\n",
```

Comment:
```suggestion
    "Quantum commuting is all about probabilities. Because our measurement only yield results in a form of zeros and ones, and each measurement collapses the quantum state, we have to run the circuit multiple times.\n",
```